.github/workflows/lint.yml

```
@@ -62,4 +62,4 @@ jobs:
         run:            cargo install cargo-spellcheck
 
       - name:           Run spellcheck
-        run:            cargo spellcheck check -m 1 -vv $(find modules/currency-exchange/src -name "*.rs")
+        run:            cargo spellcheck check -m 1 -vv $(find modules/dispatch/src -name "*.rs")
```

```suggestion
        run:            cargo spellcheck check -m 1 -vv $(find modules/dispatch/src -name "*.rs") $(find modules/currency-exchange/src -name "*.rs")
```

We should probably keep the old checks as well to make sure we don't introduce new typos.

---

In the future just add new `find` command? Thanks for the hint!